src/install/aim/agent-lang/node-config.mdx

```diff
@@ -0,0 +1,63 @@
+---
+componentType: default
+headingText: Configure the Node agent
+---
+
+You installed a Node agent in the last step, but your agent hasn't been configured to collect AI data like LLM events. This step has you apply those configurations. For AI data, you need to enable the `ai_monitoring` flag.
```

Comment:
@RyanKadri How does ai_monitoring.streaming fit into the general ai_monitoring feature flag? What does `streaming` do and what happens if a customer opts out of it? (Specifically: what sorts of data do they miss? How will the agent behave with their agent w/o the streaming part?)

Reply:
This was originally going to be a feature flag to opt out of instrumenting the streaming APIs for the LLMs. It sounds like the agent teams are no longer planning to implement this (@hmstepanek is python still thinking to keep a "streaming enabled" flag?)

If we had implemented this, it would have meant that streaming calls would not have collected any data (message content, tokens, etc). The original concern was customers wanting to avoid instrumenting streaming to reduce memory usage